lib/math.js

```diff
@@ -1,37 +1,12 @@
 /*
-* Shortens a number to the abbreviated thousands, millions, billions, etc
-* https://stackoverflow.com/a/40724354
-
-* @param {number} number: value to shorten
-* @returns {string|number}
-
-* @example
-* // return '12.3k'
-* abbreviateNumber(12345, 1)
-*/
-
-const SI_PREFIXES = ['', 'k', 'M', 'G', 'T', 'P', 'E'];
-
-export const abbreviateNumber = (number, precision = 0) => {
-  // what tier? (determines SI prefix)
-  const tier = (Math.log10(number) / 3) | 0;
-
-  const round = value => {
-    return precision === 0 ? Math.round(value) : value.toFixed(precision);
-  };
-
-  // if zero, we don't need a prefix
-  if (tier === 0) {
-    return round(number);
+ * Adds commas to separate the digits of numbers (example: 100,000)
+ */
+export const formatCurrencyWithCommas = number => {
```

Comment:
The function name could be improved since the presence of commas depends on the language. With [many languages](https://docs.oracle.com/cd/E19455-01/806-0169/overview-9/index.html), it's actually the opposite: `100.000,00`

```suggestion
export const formatCurrencyWithSeparators = number => {
```

Reply:
Done. 👍🏽